friendshipper/src/lib/components/preferences/PreferencesModal.svelte

```diff
@@ -221,6 +228,7 @@
 			showProgressModal = true;
 			progressModalTitle = 'Resetting repo...';
 			await resetRepo();
+			$repoStatus = await getRepoStatus();
```

Comment:
i noticed resetting the repo to main doesn't also update the status, so snuck that in here